src/main/java/com/ciandt/techgallery/service/impl/CronServiceImpl.java

```diff
@@ -82,28 +101,60 @@ public void sendEmailtoFollowers() {
               cal.add(Calendar.DAY_OF_MONTH, -1);
               lastCronJobExecDate = cal.getTime();
             }
-            String dailyRecommendationsIds =
-                technologyRecommendationDao
-                    .findAllRecommendationsIdsStartingFrom(lastCronJobExecDate);
-            String dailyCommentsIds =
-                technologyCommentDao.findAllCommentsIdsStartingFrom(lastCronJobExecDate);
-            List<Ref<TechGalleryUser>> followers = technologyFollowers.getFollowers();
-            if (!dailyRecommendationsIds.isEmpty() || !dailyCommentsIds.isEmpty()) {
-              for (Ref<TechGalleryUser> ref : followers) {
-                TechGalleryUser follower = ref.get();
-                emailService.push(follower, technology, dailyRecommendationsIds, dailyCommentsIds);
+
+            // Find new Activities in a technology
+            List<TechnologyRecommendation> dailyRecommendations =
+                technologyRecommendationDao.findAllRecommendationsStartingFrom(technology,
+                    lastCronJobExecDate);
+            List<TechnologyComment> dailyComments =
+                technologyCommentDao.findAllCommentsStartingFrom(technology, lastCronJobExecDate);
+            
+            // Remove Recommendations' comments. For avoid duplication
+            if (dailyRecommendations != null) {
+              for (TechnologyRecommendation recommendation : dailyRecommendations) {
+                if (dailyComments != null) {
+                  dailyComments.remove(recommendation.getComment().get());
+                }
               }
             }
+            
+            if (dailyRecommendations != null || dailyComments != null) {
+              // Create a TO to each technology
+              TechnologyActivitiesTO techActivitiesTo =
+                  createTechnologyActivitiesTo(technology, dailyRecommendations, dailyComments);
+              techActivitiesToList.add(techActivitiesTo);
+            }
+          }
+          techGalleryActivitiesTo.setTechnologyActivitiesTo(techActivitiesToList);
+
+          EmailConfig email =
```

Comment:
@brunoliberal put this block in a method in EmailService. Something like 'pushEmailConfig'